templates/arm/deployment.json.erb

```diff
@@ -173,7 +181,24 @@
               "direction": "Inbound"
             }
           }
+          <% if has_tcp_endpoints %>
+            ,
+            {
+              "name": "custom_tcp_rule",
+              "properties": {
+                "description": "Custom opened ports.",
+                "protocol": "Tcp",
+                "sourcePortRange": "*",
+                "destinationPortRange": "[parameters('tcp_endpoints')]",
+                "sourceAddressPrefix": "*",
+                "destinationAddressPrefix": "*",
+                "access": "Allow",
+                "priority": 133,
+                "direction": "Inbound"
+              }
+            }
           <% end %>
+        <% end %>
```

Comment:
Looks like has_tcp_endpoints is only being applied to Linux VMs, or are my eyes playing tricks on me?